kafka/consumerHandler_test.go

```diff
@@ -0,0 +1,116 @@
+// Package kafka provides producer and consumer to work with kafka topics
+package kafka
+
+import (
+	"encoding/json"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	saramaMocks "github.com/Shopify/sarama/mocks"
+	"github.com/google/uuid"
+	"github.com/lvl484/positioning-filter/matcher"
+	"github.com/lvl484/positioning-filter/position"
+	"github.com/lvl484/positioning-filter/repository"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewConsumerGroupHandler(t *testing.T) {
+	db, _, err := sqlmock.New()
+	assert.Nil(t, err)
+
+	filters := repository.NewFiltersRepository(db)
+	matcher := matcher.NewMatcher(filters)
+	saramaProducer := saramaMocks.NewSyncProducer(t, nil)
+	producer := &producer{
+		KafkaProducer: saramaProducer,
+	}
+	consumerGroupHandler := newConsumerGroupHandler(matcher, producer)
+	assert.NotNil(t, consumerGroupHandler.controller.matcher)
+	assert.NotNil(t, consumerGroupHandler.controller.producer)
+}
+
+func TestMessageControllerHandleMessageSuccess(t *testing.T) {
+	db, _, err := sqlmock.New()
+	assert.Nil(t, err)
+
+	filters := repository.NewFiltersRepository(db)
+	matcher := matcher.NewMatcher(filters)
+	saramaProducer := saramaMocks.NewSyncProducer(t, nil)
+	producer := &producer{
+		KafkaProducer: saramaProducer,
+		Config: &Config{
+			ProducerTopic: "filtered-positions",
+		},
+	}
+	consumerGroupHandler := newConsumerGroupHandler(matcher, producer)
+
+	var latitude float32 = 1.666
+
+	var longitude float32 = 1.333
+
+	pos := position.Position{
+		UserID:    uuid.New(),
+		Latitude:  latitude,
+		Longitude: longitude,
+		Timestamp: time.Now(),
+		Arrival:   time.Now().Add(time.Second),
+	}
+
+	b1, err := json.Marshal(pos)
+	assert.Nil(t, err)
```

Comment:
you can use `assert.NoError` in this case

Reply:
fixed